composer.json

```diff
@@ -31,7 +31,7 @@
         "drupal/m4032404": "^1.0-alpha4",
         "drupal/maxlength": "^1.0-beta2",
         "drupal/media_entity_audio": "^2.0-alpha1",
-        "drupal/metatag": "^1.7",
+        "drupal/metatag": "dev-1.x",
```

Comment:
@sonnykt , any chance of using patches rather than dev.

Reply:
@kurtfoster we can downgrade it back to 1.7. The changes are actually in the patch file https://www.drupal.org/files/issues/2019-01-18/computed_field_2945817_61.patch and it can be applied against 8.x-1.7.

Reply:
@sonnykt , I think the preference would be to stick with full versions?

Reply:
@kurtfoster I have reverted it back to 1.7.